client/extensions/wp-job-manager/components/setup/style.scss

```diff
@@ -0,0 +1,25 @@
+.wp-job-manager__setup .intro__skip-setup {
```

Comment:
Oh interesting. Should we be namespacing our component classes for extensions?

Normally, I would recommend shortening this to the component class `.intro__skip-setup`

Reply:
When I'd tested this, I was able to use a CSS class defined by another extension inside my own extension, and the styles were applied. So this seems like a "better safe than sorry" sort of thing.

Reply:
Oh I was just thinking outloud for a more formal convention. For example, letting a plugin choose a namespace then making it easy to stick to. eg with a lint error or utilities to prefix added classes.

For example maybe:
```
.wpjm_intro__skip-setup
```